src/javaRosa.js

```diff
@@ -1023,15 +1009,14 @@ define([
         if (!widget.isDefaultLang) {
             widget.mug.on('defaultLanguage-itext-changed', function (e) {
                 if (e.form === widget.form && e.itextType === widget.itextType) {
-                    var placeholder = e.value;
-                    if (!placeholder && widget.hasNodeIdPlaceholder) {
-                        placeholder = widget.mug.p.nodeID;
+                    var value = e.value;
+                    if (!_.isString(value) && widget.hasNodeIdAsDefault) {
```

Comment:
This gets hit in phantomjs, but not chrome/firefox. So I'm leaving it in here, but added a comment about why it's there.